console-subscriber/src/builder.rs

```diff
@@ -101,4 +104,41 @@ impl Builder {
     pub fn build(self) -> (TasksLayer, Server) {
         TasksLayer::build(self)
     }
+
+    /// Configures this builder from a standard set of environment variables:
+    ///
+    /// * `TOKIO_CONSOLE_RETENTION`: The number of seconds to accumulate
+    ///   completed tracing data. Default: 60s.
```

Comment:
the default retention is 60 _minutes_ (1 hour), not 60 seconds: https://github.com/tokio-rs/console/blob/103021e5dfb7d5a25381b2edbe006e9e85a676a9/console-subscriber/src/lib.rs#L196-L197

we should probably fix the docs:
```suggestion
    ///   completed tracing data. Default: 3600s (1h).
```